k9mail/src/main/java/com/fsck/k9/mailstore/MessageViewInfoExtractor.java

```diff
@@ -20,6 +19,7 @@
 import com.fsck.k9.mail.internet.Viewable;
 import com.fsck.k9.message.extractors.AttachmentInfoExtractor;
 import com.fsck.k9.ui.crypto.MessageCryptoAnnotations;
+import com.fsck.k9.ui.crypto.MessageCryptoSplitter;
 import com.fsck.k9.ui.crypto.MessageCryptoSplitter.CryptoMessageParts;
```

Comment:
Yeah I think that was from back when all of the crypto handling lived in the ui parts. Moving stuff around is no big deal, feel free